tests/ssl-opt.sh

```diff
@@ -13876,15 +13882,124 @@ run_test    "TLS 1.2 ClientHello indicating support for deflate compression meth
 
 # Most test cases are in opt-testcases/handshake-generated.sh
 
-requires_config_enabled MBEDTLS_SSL_PROTO_TLS1_2
+# Test Server Buffer resizing with fragmented handshake on TLS1.2
+# TODO investigate a a GNUTLS version for  -maxfraglen 512
+requires_openssl_3_x
+requires_protocol_version tls12
+requires_certificate_authentication
+requires_config_enabled MBEDTLS_SSL_MAX_FRAGMENT_LENGTH
+requires_config_enabled MBEDTLS_SSL_VARIABLE_BUFFER_LENGTH
+requires_max_content_len 1025
+run_test    "Handshake defragmentation on Server with buffer resizing: len=256, MFL=1024" \
+            "$P_SRV debug_level=4 auth_mode=required" \
+            "$O_NEXT_CLI -tls1_2 -split_send_frag 256 -maxfraglen 1024 -cert $DATA_FILES_PATH/server5.crt -key $DATA_FILES_PATH/server5.key" \
+            0 \
+            -s "Reallocating in_buf" \
+            -s "Reallocating out_buf" \
+            -s "reassembled record" \
+            -s "initial handshake fragment: 256, 0..256 of [0-9]\\+" \
+            -s "Prepare: waiting for more handshake fragments 256/[0-9]\\+" \
+            -s "Consume: waiting for more handshake fragments 256/[0-9]\\+"
+
+# Test Client initialised renegotiation with fragmented handshake on TLS1.2
+requires_openssl_3_x
+requires_protocol_version tls12
 requires_certificate_authentication
-run_test    "Handshake defragmentation on server: len=32, TLS 1.2 ClientHello" \
```

Comment:
Blocker: the test case "Handshake defragmentation on server: len=32, TLS 1.2 ClientHello" is still needed.

Perhaps with a tweak to the name to clarify that it's a bad case (unsupported).

Reply:
I assumed that was accidentaly left over by migration to the scripted generation. Will re-add as requested